src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -216,6 +412,13 @@ public sealed partial class JsonSerializerOptions
         public bool WriteIndented { get { throw null; } set { } }
         public System.Text.Json.Serialization.JsonConverter GetConverter(System.Type typeToConvert) { throw null; }
     }
+    public partial class JsonString : System.Text.Json.JsonNode
+    {
+        public JsonString() { }
+        public JsonString(string value) { }
+        public string GetString() { throw null; }
```

Comment:
JsonNumber has TryGet-ers as well, would it make sense to add it here or not really?

Reply:
String's get can't fail, it is-or-isn't a String.  But a JSON number can be outside the range of Int32, so GetInt32() can fail.